client/src/components/PaymentForm/PaymentForm.tsx

```diff
@@ -39,45 +43,52 @@ const PaymentForm: FC = (): JSX.Element => {
   const classes = useStyles();
   const stripe = useStripe();
   const elements = useElements();
+  const { updateSnackBarMessage } = useSnackBar();
+  const { loggedInUser } = useContext(AuthContext);
 
-  const handleSubmit = (
+  const handleSubmit = async (
     { name, line1, city, state, postal_code }: IPayment,
     { setSubmitting }: FormikHelpers<IPayment>,
   ) => {
-    // disables submission while stripe / elements haven't been loaded
     if (!stripe || !elements) {
       return;
     }
     const cardNumberElement = elements.getElement(CardNumberElement);
-    const cardExpiryElement = elements.getElement(CardExpiryElement);
-    const cardCvcElement = elements.getElement(CardCvcElement);
-    if (!cardNumberElement || !cardExpiryElement || !cardCvcElement) {
+    if (!cardNumberElement) {
       return;
     }
 
-    // need an async function to query Stripe
-    // leaving this snippet in hopefully as a hint
-    // const response = await fetch('/payments/secret');
-    // const { client_secret: clientSecret } = await response.json();
-    // const payload = await stripe.confirmCardSetup(clientSecret, {
-    //   payment_method: {
-    //     card: cardNumberElement,
-    //     billing_details: {
-    //       name: name,
-    //       address: {
-    //         line1,
-    //         city,
-    //         state,
-    //         postal_code,
-    //       },
-    //     },
-    //   },
-    // });
-    // console.log('[PaymentMethod]', payload);
+    const response = await fetch('/payments/secret');
+    const { client_secret: clientSecret } = await response.json();
+    const result = await stripe.confirmCardSetup(clientSecret, {
```

Comment:
As discussed on our call today, I'm, not sure if we are creating a customer as I don't see the call to that specific API endpoint. 

Just adding this here for additional reference: https://stripe.com/docs/payments/save-and-reuse?platform=web

Reply:
I think that we ended up with some overlap during dev.  Jeri is creating the user on stripe and prepopulating a setupintent in the User document.  I use her `const response = await fetch('/payments/secret');`  which then uses up that intent and returns the secret.  This does limit us to one payment method.  Her and I can probably talk about this next week and sort out the flow.

Reply:
Hello, this is the part where the customer is created. I think I will have to make a separate controller for the customer creation so we could have multiple setupIntents. 
https://github.com/hatchways/team-old-fashioned/blob/9d51fb075d2a1ada9c71d10c86d49760ffcff3ed/server/controllers/payment.js#L6-L19

Reply:
This will still have to be called on the front end though. Yup, let's discuss next week!